lib/membrane/rtcp/parser.ex

```diff
@@ -15,13 +15,23 @@ defmodule Membrane.RTCP.Parser do
     demand_mode: :auto
 
   def_output_pad :output, caps: RTCP, demand_mode: :auto
-  def_output_pad :rtcp_output, mode: :push, caps: :any
+
+  def_output_pad :receiver_report_output,
+    mode: :push,
+    caps: {RemoteStream, type: :packetized, content_format: one_of([nil, RTCP])}
```

Comment:
As we disscussed, I would write just `RTCP`.

Reply:
done